layers/+lang/python/packages.el

```diff
@@ -95,6 +96,19 @@
       (add-to-list 'spacemacs-jump-handlers-python-mode
                    '(anaconda-mode-find-definitions :async t)))))
 
+(defun python/init-code-cells ()
+  (use-package code-cells
+    :if (not (configuration-layer/layer-used-p 'ipython-notebook))
```

Comment:
Can you make sure that code-cells is not owned by a different layer already? Packages cannot be owned by multiple layers, if this is the case the package must be moved to one of both and this layer should then depend on the other layer via a layers.el file and configure the package using a pre/post-init hook.

Reply:
There doesn't seem to be any package which own code-cells :+1: